File: Chapitre_3/exo3_1.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc,char* argv[])
{
    int a,b;
    if(argc<=2) {
        printf("Entrez deux arguments (qui sont des nombres entiers) en lignes de commande\n");
        exit(1);
    }
    a=atoi(argv[1]);
    b=atoi(argv[2]);
    printf("Somme de %d et %d : %d\n",a,b,a+b);
    return(0);
}

File: Chapitre_3/exo3_1.py
import sys
import os

if (len(sys.argv)<=2):
    print("Entrez deux arguments (qui sont des nombres entiers) en lignes de commande")
    os._exit(1)
a=int(sys.argv[1])
b=int(sys.argv[2])
print("Somme de {} et {} : {}".format(a,b,a+b))
